### PR TITLE
Create Sprite - Usage example

### PR DESCRIPTION
**Pull request by kay-kaushik**
# Description
This pull request features usage exaples for create_sprite(). The usage example is clear and concise to help the user understand how to implement the function.

# Features
+ clear and concise code
+ gifs
+ images
+ perfect example

# Changes
4 files and 1 zip folder have been added. 2 code, 1 text and a png image.

# Benefits
These high quality usage examples give more context to the functions. This will help users reading the documentation to better understand the function. This will result in faster development.

# testing
All the changes have been tested on Kartik Kaushik's system

# Testing Images
Screenshot 2024-08-23 140643

Screenshot 2024-08-23 140653

### CODE REVIEW

File: src/assets/usage-examples-code/sprites/create_sprite/create_sprite-1-create-sprite.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+    usage example: creating a sprite
+*/
+
+using System;
+using SplashKitSDK;
+
+public class Program
```
Comment (breezy-codes) [inline src/assets/usage-examples-code/sprites/create_sprite/create_sprite-1-create-sprite.cs:8]: Update the C# program code to use top level statements only.

**Review by breezy-codes — CHANGES_REQUESTED** [review]
Make sure to update program code, and add in the png of the bitmap used into the public/usage-examples-images-gifs/sprits

File: src/assets/usage-examples-code/sprites/create_sprite/create_sprite-1-create-sprite.cs (hunk 1 above)
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/create_sprite/create_sprite-1-create-sprite.cs]: HI @breezy-codes I have commited the changes suggested by you.

**Comment by Srainyyyy** [comment]
I have tested your code, below is a screenshot of the test
c++
image
c#
image

**Review by Srainyyyy — COMMENTED** [review]
It looks pretty good, C# is the top level language, and I can also find player.png in the images folder

**Review by breezy-codes — APPROVED** [review]
looks good now

**Review by NidhishaPahade — APPROVED** [review]
Everything looks good and works as expected.

Screenshots of the code running successfully:

Screenshot 2024-09-25 220503

Screenshot 2024-09-25 220422

**Pull request merged**